src/main/java/ua/com/javarush/gnew/crypt/code/RunBruteforce.java

```diff
@@ -0,0 +1,83 @@
+package ua.com.javarush.gnew.crypt.code;
+
+
+import ua.com.javarush.gnew.Constants.Constants;
+
+
+public class RunBruteforce {
```

Comment:
Very good approach here. And my suggestion is the same here. It might be simplified by extracting methods and removing duplicates.
https://refactoring.guru/extract-method
https://refactoring.guru/uk/smells/duplicate-code